.github/workflows/pytorch-linux-xenial-py3.6-gcc5.4.yml

```diff
@@ -142,8 +142,17 @@ jobs:
       - name: Archive artifacts into zip
         run: |
           zip -r artifacts.zip dist/ build/
+      # Upload to github so that people can click and download artifacts
       - uses: actions/upload-artifact@v2
```

Comment:
if this step is fyi only, maybe we can add `continue-on-error: true` to bypass its flakiness

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

Reply:
Done